### ISSUE TITLE
New logo

### ISSUE THREAD

**Issue by veitveit** [bio-tools/biotoolsRegistry#588]
@hmenager @magnuspalmblad @matuskalas 

I think bio.tools needs a new logo. Any ideas or should I make a few suggetions?

**Comment by hmenager** [comment]
Agree!

**Comment by veitveit** [comment]
How about something like this?
![image](https://github.com/bio-tools/biotoolsRegistry/assets/15800709/4252581c-db1b-4f26-b7ba-fddceeb173fb)

**Comment by magnuspalmblad** [comment]
Looks nice, but none of the ELIXIR Core Data Resources have the ELIXIR DNA thingie in their logos. Quite a few have the name of the resource in it, so I think that is fine. Could we make it a simple toolbox with bio.tools written on the side and two tools in it?

**Comment by veitveit** [comment]
As the name is the same as the URL, I prefer to keep it in. I did not find any free picture/svg of a suitable toolbox, anyone has something?
And I am a bit divided on the ELIXIR "antenna". Would be nice to have something pointing to ELIXIR. On the other hand, the colors are from the ELIXIR palette.

**Comment by magnuspalmblad** [comment]
I agree 100% with keeping the name=URL in. However, I assume the bio.tools logo will occasionally be displayed together with the ELIXIR (Denmark) logo. Including the ELIXIR "antenna" may contradict the spirit of the ELIXIR brand guidelines (https://elixir-europe.org/system/files/elixir_brand_guidelines_2013.pdf) as these do not allow the use of two versions of the ELIXIR logo next to each other ("In the interests of maximising the impact of the ELIXIR brand, the main ELIXIR logo must not be used directly next to the ELIXIR node logo. The overall effect of this juxtaposition is confusing and weakens the impact of both logos.". When I am back at work, I could make something simple based on a toolbox, with 2-3 tools and "bio.tools" written along the side.

Something along these lines (courtesy of DALL-E):

![toolbox](https://github.com/bio-tools/biotoolsRegistry/assets/16084125/c8f6c288-74e2-4fae-8a78-3ecac85af96a)
![image](https://github.com/bio-tools/biotoolsRegistry/assets/16084125/46545d1a-ab72-45ab-a741-51cc189d30bd)

**Comment by veitveit** [comment]
Looks good! But we will need this as svg. Here is my svg, now without the ELIXIR thing. I did not see this part of the guidelines.

![New Logo](https://github.com/bio-tools/biotoolsRegistry/assets/15800709/103df4be-6d21-4687-b53d-e4c7154dfd8f)

**Comment by magnuspalmblad** [comment]
Oh, I was not suggesting we take something generated by an AI - I just used them to illustrate the idea. I would make it in Illustrator (can be exported to SVG). There is no hurry with this, right? We could create a few examples, discuss and refine them in the coming weeks?

**Comment by veitveit** [comment]
No hurry at all. Thanks for helping out!

**Comment by veitveit** [comment]
Also now added a favicon. Comments are welcome.
https://raw.githubusercontent.com/bio-tools/biotoolsRegistry/refs/heads/master/frontend/img/logos/NewBioToolsFavicon.svg
